src/components/About/AboutUIModal.tsx

```diff
@@ -31,13 +31,7 @@ class AboutUIModal extends React.Component<AboutUIModalProps, AboutUIModalState>
 
   render() {
     return (
-      <AboutModal
-        show={this.state.showModal}
-        onHide={this.close}
-        productTitle="Kiali"
-        logo={pfLogo}
-        altLogo="Kiali Logo"
-      >
+      <AboutModal show={this.state.showModal} onHide={this.close} productTitle={<img src={pfLogo} />}>
```

Comment:
Instead of using the pfLogo, I'd suggest using the same color that you are using for the background of the Login page.